modules/gitops-creating-rolloutmanager-custom-resource.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-creating-rolloutmanager-custom-resource_{context}"]
+= Creating RolloutManager custom resource
+
+To manage progressive delivery of deployments by using the Argo Rollouts feature in {gitops-title}, you must create and configure a RolloutManager custom resource (CR) in the namespace of your choice. By default, any new `argo-rollouts` instance has permission to manage resources only in the namespace where it is deployed. But you can use Argo Rollouts in multiple namespaces as required.
+
+.Prerequisites
+
+* {gitops-title} 1.9.0, or a later version, is installed in your cluster.
+
+.Procedure 
+
+. Log in to the {product-title} web console. 
```

Comment:
Maybe mention they need to use the Administrator perspective since that's where Operators are managed. They probably also need cluster admin permissions for this procedure? I'd check with engineering.